.github/workflows/release.yml

```diff
@@ -0,0 +1,38 @@
+on:
+  workflow_call:
+    inputs:
+      environment:
+        required: true
+        description: Environment name
+        type: string
+jobs:
+  deploy:
+    name: Deploy
+    runs-on: ubuntu-latest
+    environment:
+      name: ${{ inputs.environment }}
+    steps:
+        # v4.1.1
+      - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11
```

Comment:
Is it necessary to checkout or install Node.js deps here? It seems like we don't use the repo code itself or any dependencies

Reply:
I ask because we decided to not bring in the repo or third-party dependencies at all in the publish step for our libraries, just to minimize the chances that any secrets in our production environment would be exfiltrated by a malicious dependency